server/src/core/auth/strategies/google.auth.strategy.ts

```diff
@@ -16,10 +16,17 @@ export type GoogleRequest = Request & {
 @Injectable()
 export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
   constructor(environmentService: EnvironmentService) {
+    const isAuthGoogleEnabled = environmentService.getAuthGoogleEnabled();
     super({
-      clientID: environmentService.getAuthGoogleClientId(),
-      clientSecret: environmentService.getAuthGoogleClientSecret(),
-      callbackURL: environmentService.getAuthGoogleCallbackUrl(),
+      clientID: isAuthGoogleEnabled
```

Comment:
@magrinj @emilienchvt not very proud of these disabled but `super() `has to be called.
Another solution would be to not register the strategy using DynamicModules

Reply:
Not extremely better, but disabledClientId, disabledClientSecret and disabledCallbackUrl might be less generic